### DISCUSSION TITLE
Choosing start position for circular scaffold

### DISCUSSION THREAD

**Discussion by jmhallas** [Kinggerm/GetOrganelle#230] [Q&A] [answered]
Hi,

I was wondering if there is a way to choose a specific gene to start the completed mtDNA genome from?  For example, start the scaffold at tRNA-Phe.  In the scaffold getorganelle assembled for me tRNA-Phe is at position 11,175, and I would like it to be at position 1.

Thank you for your time and suggestions.

-Josh Hallas

**Comment by JianjunJin** [comment]
Hi, thanks for reaching out. I'm sorry that we don't have a function for reseeding a circular sequence currently, which may involve accurate annotation.

**Comment by jmhallas** [comment]
Hello,

I don't know what you mean by reseeding?  I am using a complete mtDNA
genome with an annotation as a seed but the seed starts at different gene
than my assembled mtDNA from getorganelle.  Is there a way to get my new
assembly to start at the same gene?  This is the command I'm using

get_organelle_from_reads.py -1 DS18041_R1.fastq.gz -2 DS18041_R2.fastq.gz
-k 21,45,65,85 -s Hassanin_2012_JN632670.fasta --genes gene.fasta -F
animal_mt -t 32 -o odocoileous_mitogenomes_4

**Reply by JianjunJin** [reply]
Sorry about the unclear of what I meant by using an improper word reseed - I meant to restart a circular sequence.
I totally got your point. The clear answer should be there is currently no way of doing that automatically through GetOrganelle.

-
I have been thinking about that because there are many demands, including getting a new start for a circular genome and picking the structure looking more like the reference when there are multiple structures. But I can only work on that after pushing on other issues. It is more like a downstream treatment but will be useful.  I can keep you posted if it is available, but it will be a while.

**Comment by jmhallas** [comment, marked as answer]
Thank you for clarifying.  It would be a nice feature.  I was hoping I just
might have missed something in the getorganelle manual.

Thanks again.
